### DISCUSSION TITLE
Where do I find the default config for the hosted github app?

### DISCUSSION THREAD

**Discussion by wwuck** [renovatebot/renovate#23423] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I'm trying to debug a renovate problem on my self-hosted gitlab instance where changelogs are not appearing for all MR.  I decided to run a test on the hosted github app with a test repository and all is working properly there.  My conclusion is that there could be a config difference between my local self-hosted config and the default config on github where I didn't set any extra config.  Where can I find the default config for the hosted github app so I can compare to my local config?

### What you tried so far.

I searched all the renovate docs and github search but couldn't find anything.

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
It's printed to the logs available at develop.mend.io
